src/js/state/migrations/202006091248_moveSidebarViewToTabLayout.test.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+
+import getTestState from "../../test/helpers/getTestState"
+import migrate from "./202006091248_moveSidebarViewToTabLayout"
+
+test("migrating 202006091248_moveSidebarViewToTabLayout", () => {
+  let prev = getTestState("v0.9.1")
```

Comment:
I think this works. But ideally we'd be testing against the previous released version. So we'd make one called v0.10.0. This can be done in the "Developer" menu when the app is running.